.github/codeql-config.yml

```diff
@@ -0,0 +1,4 @@
+name: "CodeQL config"
```

Comment:
<p>Why not put it in the root path? then it is less configuration in the action itself</p>

Reply:
as far as I know, CodeQL only checks in the user folder for a possible config file. And in the officical docs they also reference a config file inside the `.github` folder, but even more nested under `codeql`